packages/components/src/internal/components/picklist/PicklistOverview.tsx

```diff
@@ -331,7 +331,7 @@ export const PicklistOverview: FC<OwnProps> = memo(props => {
         <PicklistOverviewWithQueryModels
             {...props}
             key={LOAD_PICKLIST_COUNTER}
-            autoLoad={false}
+            autoLoad
```

Comment:
are all of the usages of the sample tabbed grid switching to autoLoad for showing the tab row counts? or was this change for some other reason?